packages/contracts/hardhat.solidity-config.ts

```diff
@@ -6,7 +6,7 @@ export const solidityConfig = {
       settings: {
         optimizer: {
           enabled: true,
-          runs: 25,
+          runs: 10,
```

Comment:
would be great to capture any affect this has on mint gas performance by checking with/without the change using `test/core/gas-tests/GenArt721CoreV3_GasTests_Mint.test.ts`, and noting in PR description.

Reply:
Great call! 🙏 Updated the description 🙏